### ISSUE TITLE
Publish to rubygems

### ISSUE THREAD

**Issue by shockie** [fabn/activeadmin-globalize#16]
Is it possible to release a version to rubygems.org? This will enable other gems to depend on the project. The original 'activeadmin-globalize' isn't maintained anymore perhaps namespace the gem as 'activeadmin-globalize-fabn' ?

**Comment by micred** [comment]
Looks like even this fork is not maintained anymore, looking to other issues like #5 this gem require gimmicks to work with AA 1.0.0 / Rails 5.

**Comment by fabn** [comment]
@shockie @micred This gem **is** published to rubygems, you can find [it here](https://rubygems.org/gems/activeadmin-globalize), however it's only published for version 0.6.x of activeadmin.

The main issue in keeping that up to date is that nor ActiveAdmin neither Globalize have properly release policies and it's very hard to follow and fix all edge cases coming from following their master branches.

I'm not criticizing those projects, they both are awesome projects and I've used them for years, but in this moment I do not have any active projects using these technologies, so I'm not able to work to this.

If/when they comes out with stable versions released on rubygems I'll try to do the same, but currently it's near impossible to follow and fix all the issues coming out in all different master versions.

**Comment by mauriciopasquier** [comment]
AA finally realeased [1.0.0](https://rubygems.org/gems/activeadmin/versions/1.0.0), what can we do to help release this gem now?

**Comment by mauriciopasquier** [comment]
Awesome, thanks!
